### PR TITLE
add quant positions for S22

### PR DESCRIPTION
**Pull request by YangF0917**
added a few quant positions, can also link quant new grads but not sure since they're not internships

### CODE REVIEW

**Comment by Aaronchoo** [comment]
LGTM

**Review by Aaronchoo — APPROVED** [review]

**Review by kallentu — CHANGES_REQUESTED** [review]
Can we make sure these internships are available for Winter? I wouldn't want to be misleading people who are looking solely for winter roles. The only one here that seems like a winter role is Citadel, the other ones look like summer only?

**Comment by YangF0917** [comment]
Ahhh my bad will add them to S22 repo when/if it's made

File: README.md [hunk 1]
```diff
@@ -36,11 +36,13 @@ Good luck and happy hunting :tada:!
...
+| [NVIDIA](https://nvidia.wd5.myworkdayjobs.com/UniversityJobs/11/refreshFacet/318c8bb6f553100021d223d9780d30be) | Multiple Positions | Santa Clara, Remote | For Fall 2021, Winter and Summer 2022. |
 | [Shopify](https://www.shopify.ca/careers/search?teams%5B%5D=interns&keywords=&sort=team_asc) | Various (SWE, Data Sci, Data Eng) | Remote (North America) | |
 | [Jane Street](https://www.janestreet.com/join-jane-street/position/5339546002/) | SWE | New York | Visa support available. |
 | [RBC](https://jobs.rbc.com/ca/en/job/404796/Software-Developer-Winter-Student-2022-Opportunities) | Software Developer | Remote | Potential to work in person depending on Covid restrictions |
 | [IBM](https://www.ibm.com/employment/#jobs?&job-search=software%20developer%20intern%204%208%20months) | Software Developer | Ottawa, ON or Markham, ON | 4 or 8 Months |
+| [Citadel Securities](https://www.citadelsecurities.com/careers/details/trader-intern-us/) | SWE, Quant Trader/Researcher | NYC, Chicago | 11 weeks, Research open to PhD students only |
```
Comment (kallentu) [inline README.md]: Is there a way to link the intern search? So we can lead people to all of the jobs instead of just the trader job?

Comment (YangF0917) [reply README.md]: Hmm actually the SWE position is the same as Citadel's above, so I'll just merge them above

**Review by kallentu — APPROVED** [review]

**Pull request merged**